### PR TITLE
Styles/theme

### PR DESCRIPTION
**Pull request by Ruben9933**

### CODE REVIEW

File: client/src/appTheme.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      main: "#27EA8C",
+      light: "#7CF8A3",
+      dark: "#1CC987",
+    },
+  },
+
+  background: {
+    default: "#F3FEFE",
+    secondary: "#F0FBFB",
+    paper: "#FFF",
```
Comment (devBritt) [inline client/src/appTheme.js:45]: Add another background color option `dark: "#2D334A`

File: client/src/components/footer/index.js [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
-            component={'p'}
-            variant={'p'}
-            textAlign={'left'}
-            sx={{ flexGrow: 1 }}
+    <Toolbar sx={{ bgcolor: "#2D334A" }}>
+      <Typography
+        component={"p"}
+        variant={"p"}
+        textAlign={"left"}
+        sx={{ flexGrow: 1, color: "#ADBECC" }}
```
Comment (devBritt) [inline client/src/components/footer/index.js:10]: replace `color: "#ADBECC"` with `color: "text.hint"`

File: client/src/components/footer/index.js [hunk 3]
```diff
@@ -1,41 +1,41 @@
...
-                underline={'none'}
-                target={'_blank'}
-            >
-                Ruben
-            </Link>
-        </Typography>
+          /pawsitive-trainers
+        </Link>
+      </Typography>
+      <Typography component={"p"} variant={"p"} sx={{ color: "#ADBECC" }}>
```
Comment (devBritt) [inline client/src/components/footer/index.js:21]: replace `color: "#ADBECC"` with `color: "text.hint"`

File: client/src/components/sign-up/signup.js [hunk 4]
```diff
@@ -139,6 +139,7 @@ export default function Signup() {
                                 component={'h2'}
                                 variant="h4"
                                 textAlign={'center'}
+                                sx={{ fontWeight: "bold", color: "#89D6D7" }}
```
Comment (devBritt) [inline client/src/components/sign-up/signup.js:142]: replace `color: "secondary"` with `color: "secondary"`

File: client/src/components/sign-up/signup.js [hunk 5]
```diff
@@ -157,6 +158,7 @@ export default function Signup() {
...
```
Comment (devBritt) [inline client/src/components/sign-up/signup.js:161]: replace `color: "secondary"` with `color: "secondary"`

File: client/src/pages/homepage/index.js [hunk 6]
```diff
@@ -26,7 +27,7 @@ export default function Home() {
             <Hero />
             <SearchContainer />
             <SearchResults />
-            <Container sx={{ my: 5 }}>
+            <Container sx={{ my: 5, bgcolor: "#F0FBFB" }}>
```
Comment (devBritt) [inline client/src/pages/homepage/index.js:30]: replace `color: "background.secondary"` with `color: "background.secondary"`

File: client/src/pages/trainerProfile/index.js [hunk 7]
```diff
@@ -9,7 +9,7 @@ export default function Trainer() {
         </div>
       </container>
 
-      <container maxWidth="0.5">
+      <container maxWidth="0.5" sx={{ bgcolor: "#DAF3F2 " }}>
```
Comment (devBritt) [inline client/src/pages/trainerProfile/index.js:12]: replace `color: "#DAF3F2"` with `color: "secondary.light"`

File: seed.js [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+    phone: {
+        type: Sequelize.STRING,
+        allowNull: false
+    },
+    externalLinks: {
+        type: Sequelize.STRING
+    },
+    review: {
+        type: Sequelize.STRING
+    }
```
Comment (devBritt) [inline seed.js:49]: This isn't what a seed file is supposed to be. We aren't using Sequelize in this project, either, cause Sequelize is for MySQL and we're using MongoDB. Go ahead and wipe this entire file and I'll take care of it.

**Review by devBritt — APPROVED** [review]
